regen-network/regen-web
# test: add ksh to local dev environment

## Description



This pull request adds the kilo-sheep-hour credit type, class, and project to the local dev environment setup. This pull request also updates the `CSRF` environment variables to fix logging in with Keplr when using the local dev environment.

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test


1. create a dump file for restoring the production db

```
pg_dump [DATABASE_URL] --file dump-server-production -F c
```

2. build and start the local dev environment

```
docker-compose up
```

3. start the marketplace application

```
yarn start
```

4. confirm kilo-sheep-hour credit class and project

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
